docs/conf.py

```diff
@@ -61,7 +61,7 @@
 master_doc = 'index'
 
 # General information about the project.
-project = u'F5 OpenStack LBaaSv2 Driver'
+project = u'F5 OpenStack LBaaSv2 Plugin'
```

Comment:
We are not a Plugin.  neutron_lbaas is the plugin.  We are a plugin driver, service provider, or just driver.